.devcontainer/Dockerfile

```
@@ -0,0 +1,5 @@
+# This dockerfile is used to build the devcontainer environment.
+# more info about vscode devcontainer: https://code.visualstudio.com/docs/devcontainers/containers
+FROM mcr.microsoft.com/devcontainers/python:1-3.9-bullseye
+# Need to install `liblz4-dev` `libunwind-dev` packages for ddev installation see: https://github.com/DataDog/integrations-core/issues/14043
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt install -y liblz4-dev libunwind-dev --option=Dpkg::Options::=--force-confdef
```

Let's avoid linking issues and just install what we need. Also, I don't think that last part is necessary:

```suggestion
RUN apt update && DEBIAN_FRONTEND=noninteractive apt install -y liblz4-dev libunwind-dev
```